packages/released/coq-menhirlib/coq-menhirlib.20190613/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+synopsis: "A support library for verified Coq parsers produced by Menhir"
+maintainer: "francois.pottier@inria.fr"
+authors: [
+  "Jacques-Henri Jourdan <jacques-henri.jourdan@lri.fr>"
+]
+homepage: "https://gitlab.inria.fr/fpottier/coq-menhirlib"
+dev-repo: "git+https://gitlab.inria.fr/fpottier/menhir.git"
+bug-reports: "jacques-henri.jourdan@lri.fr"
+build: [
+  [make "-C" "coq-menhirlib" "-j%{jobs}%"]
+]
+install: [
+  [make "-C" "coq-menhirlib" "install"]
+]
+remove: [
+  [make "-C" "coq-menhirlib" "uninstall"]
+]
+depends: [
+  "coq" { >= "8.6" }
+  "menhir" { = "20190613" }
+]
+url {
```

Comment:
Please add some metadata under `tags`, at least `"logpath:MenhirLib"` and the release date. See [here](https://github.com/coq/opam-coq-archive#website-and-opam-metadata) for an example.

Reply:
Done.